src/dsp_tools/commands/project/create/project_create.py

```diff
@@ -81,18 +72,21 @@ def _create_project_on_server(
     success = True
     project_local = Project(
         con=con,
-        shortcode=shortcode,
-        shortname=shortname,
-        longname=longname,
-        description=LangString(descriptions),  # type: ignore[arg-type]
-        keywords=set(keywords) if keywords else None,
+        shortcode=project_definition.shortcode,
+        shortname=project_definition.shortname,
+        longname=project_definition.longname,
+        description=LangString(project_definition.descriptions),  # type: ignore[arg-type]
+        keywords=set(project_definition.keywords) if project_definition.keywords else None,
```

Comment:
if you feel like, you could teach the constructor of `Project` how to deal with a `ProjectDefinition`

Reply:
It would be more elegant to read, but in my opinion it would increase the dependencies too much. I aimed to keep the functions as stupid as possible.